website/docs/cli/configuration.mdx

```diff
@@ -753,8 +753,8 @@ logs:
 Checking out Git ref 'refs/remotes/origin/HEAD' ...
 Checked out Git ref 'refs/remotes/origin/HEAD'
 
-Current working repo HEAD: ffd2154e1daa32357b75460b9f45d268922b51e1 refs/heads/update-logs
-Remote repo HEAD: f7aa382aa8b3d48be8f06cfdb27aad344b89aff4 HEAD
+Current HEAD: ffd2154e1daa32357b75460b9f45d268922b51e1 refs/heads/update-logs
+Base HEAD: f7aa382aa8b3d48be8f06cfdb27aad344b89aff4 HEAD
```

Comment:
I think there's some terminology confusion here.

`HEAD` (uppercase) always means the currently checked out commit
`head` (lowercase) means the tip (latest commit) of some branch (head main is the tip of main)
`ref` a human-friendly pointer to a commit, which may be a branch name (refs/heads) or a tag name (refs/tags)
`BASE` an arbitrary commit, which may or may not be a `head`, `ref` or `HEAD` to perform some action against

So in this case, we are comparing `HEAD` to a `BASE` commit that we specify to see if any changes have occurred since. `BASE`.